mlflow/utils/credentials.py

```diff
@@ -76,29 +77,25 @@ def login(backend="databricks"):
         )
 
 
-def _check_databricks_auth():
-    # Check if databricks credentials are set.
-    try:
-        from databricks.sdk import WorkspaceClient
-    except ImportError:
-        raise ImportError(
-            "Databricks SDK is not installed. To use `mlflow.login()`, please install "
-            "databricks-sdk by `pip install databricks-sdk`."
-        )
+def _validate_databricks_auth():
+    """Validate Databricks authentication."""
+    max_trials = 3  # Allow 3 retrys for timeout.
+    timeout = 3
+    for _ in range(max_trials):
+        try:
+            result = subprocess.run(
+                ["databricks", "tokens", "list"],
+                timeout=timeout,
+                stdout=subprocess.DEVNULL,
+                stderr=subprocess.DEVNULL,
+            )
+            if result.returncode != 0:
+                raise MlflowException("Invalid databricks credentials.")
```

Comment:
```suggestion
                raise MlflowException("Failed to validate databricks credentials.")
```

because `databricks tokens list` failed doesn't necessarily mean the credentials are invalid.